aptos-node/src/lib.rs

```diff
@@ -349,6 +349,9 @@ pub fn setup_environment_and_start_node(
     remote_log_rx: Option<mpsc::Receiver<TelemetryLog>>,
     logger_filter_update_job: Option<LoggerFilterUpdater>,
 ) -> anyhow::Result<AptosHandle> {
+    // We must log the node config here to ensure it is accessible in the logs
+    info!("Using node config {:?}", &node_config);
```

Comment:
shall we group it with the println! or replace it?

Reply:
yeah, why not just replace println! ?

But also what do you mean by "but this means it's not accessible in our logging infrastructure" ?

I can see it in Humio:

https://cloud.us.humio.com/mainnet/search?end=1677001344119&live=false&query=NodeConfig&start=1677000766250&tz=America%2FLos_Angeles

Reply:
Thanks folks. Yeah, if we're happy to just delete the `println!`, I'll do that 😄 

> But also what do you mean by "but this means it's not accessible in our logging infrastructure" ?

Weird... I was looking for this log line in one of your 3-region 100 node experiments and it didn't show up. So maybe we're pushing log lines differently for `println!` ? Either way, I'll change this and we can monitor going forward.